Automattic/wp-calypso
# Performance Profiler: Update Recommendations responsiveness

Fixes [Performance Profiler: Make the recommendations section responsive](https://github.com/Automattic/dotcom-forge/issues/8913)
Fixes [Performance Profiler: Make profiler responsive to various screen sizes](https://github.com/Automattic/dotcom-forge/issues/8898)

## Proposed Changes

* Make the section stop overflowing horizontally
* Update wrap of the tip section

## Why are these changes being made?
[Performance Profiler: Make the recommendations section responsive](https://github.com/Automattic/dotcom-forge/issues/8913)

## Testing Instructions


* Go to `/speed-test-tool?url=wordpress.com&flags=performance-profiler/llm` (Don't forget to add the LLM flag)
* Resize the window and check stuff is being organized correctly inside the recommendations section

![CleanShot 2024-09-05 at 17 43 11](https://github.com/user-attachments/assets/98d13e86-b3c6-4866-a0ca-ccd49bcc57af)


## Pre-merge Checklist
